drivers/adc/ad7616/ad7616.c

```diff
@@ -1050,7 +1050,7 @@ int32_t ad7616_setup(struct ad7616_dev **device,
 int ad7616_remove(struct ad7616_dev *dev)
 {
 	if (!dev)
-		return -ENODEV;
+		return;
```

Comment:
I just saw this, this change should be in the commit.

Reply:
I will revert this, my bad